libcxx/test/std/input.output/iostream.format/std.manip/setfill_eof.pass.cpp

```diff
@@ -0,0 +1,31 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Test that weof as a wchar_t value can be set as the fill character.
+
+// UNSUPPORTED: no-wide-characters
+// REQUIRES: target=powerpc{{(64)?}}-ibm-aix || target=s390x-ibm-zos
```

Comment:
I think we can expect this test to pass on all targets which don't have to opt out for ABI compat (i.e. everything but windows in v1 ABI), so this should be something like:
```suggestion
// XFAIL: target=={{.+}}-windows-{{.+}} && stdlib=libc++ && libcpp-abi-version=1
```

Reply:
Good point! Changed as suggested, thanks!